src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -68,9 +68,7 @@ const StyledListItem = styled.li`
 `;
 
 const PodcastExternalLinks = ({ brandTitle, links }) => {
-  const { translations, service, script, dir, lang } = useContext(
-    ServiceContext,
-  );
+  const { translations, service, script, dir } = useContext(ServiceContext);
```

Comment:
why not use the `lang` from the service context?

Reply:
@jroebu14 because the language of the podcast external link text may differ from the service language. For example the on https://www.test.bbc.com/marathi/podcasts/p09437hm?renderer_env=live "Spotify" needs to be read in English

Reply:
Ah I see now! Cool thanks for explaining